app/src/main/java/fr/free/nrw/commons/mwapi/UploadResult.java

```diff
@@ -36,13 +36,16 @@
 
     @Override
     public String toString() {
-        return "UploadResult{" +
-                "errorCode='" + errorCode + '\'' +
-                ", resultStatus='" + resultStatus + '\'' +
-                ", dateUploaded='" + dateUploaded.toString() + '\'' +
-                ", imageUrl='" + imageUrl + '\'' +
-                ", canonicalFilename='" + canonicalFilename + '\'' +
-                '}';
+        if (dateUploaded == null) {
```

Comment:
No not return early if date uploaded is null. 

add a check like: `dateUploaded==null? "":dateUploaded.toString()`

Reply:
I have changed the condition checking into a single line. Check the last commit.